gridcapa-cse-valid-app/src/main/java/com/farao_community/farao/cse_valid/app/service/FullImportComputationService.java

```diff
@@ -39,19 +43,28 @@
 @Service
 public class FullImportComputationService {
 
+    private final ComputationService computationService;
     private final DichotomyRunner dichotomyRunner;
     private final FileImporter fileImporter;
     private final FileExporter fileExporter;
     private final Logger businessLogger;
+    private final CseValidNetworkShifterProvider cseValidNetworkShifterProvider;
+    private final CseValidRaoValidator cseValidRaoValidator;
 
-    public FullImportComputationService(DichotomyRunner dichotomyRunner,
+    public FullImportComputationService(ComputationService computationService,
+                                        DichotomyRunner dichotomyRunner,
                                         FileImporter fileImporter,
                                         FileExporter fileExporter,
-                                        Logger businessLogger) {
+                                        Logger businessLogger,
+                                        CseValidNetworkShifterProvider cseValidNetworkShifterProvider,
+                                        CseValidRaoValidator cseValidRaoValidator) {
+        this.computationService = computationService;
         this.dichotomyRunner = dichotomyRunner;
         this.fileImporter = fileImporter;
         this.fileExporter = fileExporter;
         this.businessLogger = businessLogger;
+        this.cseValidNetworkShifterProvider = cseValidNetworkShifterProvider;
+        this.cseValidRaoValidator = cseValidRaoValidator;
     }
 
     public void computeTimestamp(TTimestampWrapper timestampWrapper, CseValidRequest cseValidRequest, TcDocumentTypeWriter tcDocumentTypeWriter) {
```

Comment:
Introduce a `timestamp` variable in the same way as it has been done in ExportCornerComputationService to avoid repeating `timestampWrapper.getTimestamp()`.